packages/core/package.json

```diff
@@ -13,7 +13,8 @@
   ],
   "contributors": [
     "Alex Sexton <alexsexton@gmail.com>",
-    "Eemeli Aro <eemeli@gmail.com>"
+    "Eemeli Aro <eemeli@gmail.com>",
+    "cdaringe<cdaringe@gmail.com>"
```

Comment:
is that bad form 😄 ?

Reply:
I don't really mind this, but please do at least match the format of the other entries.